Doxyfile

```diff
@@ -0,0 +1,2567 @@
+# Doxyfile 1.8.16
```

Comment:
This file should be `.gitignored`. Running `configure_file(${DOXYFILE_IN} ${DOXYFILE_OUT} @ONLY)` as part of build generates it from `Doxyfile.in`, so there is no need to have it checked in.